portfolio/src/main/webapp/script.js

```diff
@@ -34,23 +34,39 @@ function addRandomGreeting() {
 }
 
 function getData() {
-    console.log("We are Here;");
-    const responsePromise = fetch('/data');
+  console.log("We are Here;");
+  const responsePromise = fetch("/data");
 
-    responsePromise.then(handleResponse);
+  responsePromise.then(handleResponse);
 }
 
 function handleResponse(response) {
-  console.log('Handling the response.');
+  console.log("Handling the response.");
```

Comment:
I noticed that you are constantly changing the use of single quote to double quote. FYI, they are equivalent in terms of representing a string in MOST cases: https://staxmanade.com/2018/03/should-i-use-javascript-single-or-double-quotes/

Which one to choose? Well doesn't really matter, just keep consistent within the file/library.

Reply:
It is not me who changes the quotes, I am using the program Lyle told us about to format our code and for some reason it likes to change the quotation marks.